.github/workflows/test.yml

```diff
@@ -12,14 +12,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        swift:
-          - swift:5.4
-          - swift:5.5
-          - swift:5.6
-          - swiftlang/swift:nightly-main
-        os:
-          - focal
-    container: ${{ format('{0}-{1}', matrix.swift, matrix.os) }}
+        container:
+          - swift:5.5-bionic
```

Comment:
What is the benefit to testing on `bionic`?

Reply:
We have a nice mix of oses and swift versions...